src/routeGeneration/templates/express.ts

```diff
@@ -76,23 +76,42 @@ export function RegisterRoutes(app: any) {
         return (request: any, response: any, next: any) => {
             let responded = 0;
             let success = false;
+
+            const succeed = function(user: any) {
+                if (!success) {
+                    success = true;
+                    responded++;
+                    request['user'] = user;
+                    next();
+                }
```

Comment:
Could _responded_ and the _next_ function call be outside of the _if_ ?

Reply:
Ah, I removed the comment there. I think the idea - and I can only guess because I didn't write it - was that we're not calling next multiple times here.